### DISCUSSION TITLE
Render / route on 404

### DISCUSSION THREAD

**Discussion by mgill404** [remix-run/remix#8436] [Q&A] [answered]
Hello,

I'm working on a Shopify app and they have some unclear terms around 404 errors.

https://shopify.dev/docs/apps/store/requirements
> Operational errors (within an app's functionality) are acceptable, but web errors such as 404's, 500's and 300's are not.

I have checked some popular Shopify apps, and they handle 404 by rendering the `/` route. 

I would like to render `/` on 404 if it's supported somehow. I currently render this route with a routes/_index.jsx file.

I can't figure out how to do this in Remix. Any suggestions?

**Comment by sergiodxa** [comment, marked as answer]
Create a splat route with routes/$.tsx, export a loader where you redirect to /

**Reply by mgill404** [reply]
Excellent.
